.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
You can probably remove this file, these should go in your global gitignore file 🙂